frontend/src/features/projects/api-service.ts

```diff
@@ -0,0 +1,34 @@
+import queryString from 'query-string';
```

Comment:
Not so sure about the naming of this file, any reason why it isn't just apiService.ts?

Reply:
No reason really, I will rename it